src/test/java/duke/data/TaskListTest.java

```diff
@@ -0,0 +1,29 @@
+package duke.data;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.data.task.Todo;
+
+public class TaskListTest {
+    @Test
+    public void testAddTask() {
```

Comment:
I think you can add more details to the test method using the following convention:
whatIsBeingTested_descriptionOfTestInputs_expectedOutcome

Reply:
Yes, I will take note of this.